floop-data-export.py

```diff
@@ -9,6 +9,8 @@
     certLoc = input('Enter full path of Floop Firebase cert file: ')
     gPath = os.path.isfile(certLoc)
 
+    n = int(input('Enter the number of conversations required: '))
```

Comment:
I see the thinking behind having an option for determining a different number each time, but as we are moving to no user entry, we might consider simply hard-coding line 28's value to something less than 10,000 for the time being. We might add a command line variable next sprint for a variable number of entries, perhaps?

Reply:
Understood. Thanks Mike. Will soon create new PR with these changes incorporated